example.yaml

```diff
@@ -0,0 +1,9 @@
+# Config for node URLs
+Location: [1,2]
```

Comment:
This file is a duplicate of `config/config.yaml.dist`. Please remove this, and add a note to the README to use `config.yaml.dist` as a distributed example config.